### PR TITLE
feat: updated send flow ARC-236

### PR DESCRIPTION
**Pull request by nicholaswma**

### CODE REVIEW

File: src/routes/popup/send/auth.tsx [hunk 1]
```diff
@@ -166,74 +178,153 @@ export default function SendAuth({ tokenID }: Props) {
...
+          duration: 2000
+        });
+        uToken
+          ? push("/")
+          : push(
+              `/transaction/${transaction.id}?back=${encodeURIComponent("/")}`
+            );
+
+        // remove wallet from memory
+        freeDecryptedWallet(keyfile);
```
Comment (7i7o) [inline src/routes/popup/send/auth.tsx:317]: We are not calling `freeDecryptedWallet(keyfile)` if the tx fails

File: src/routes/popup/send/auth.tsx [hunk 2]
```diff
@@ -166,74 +178,153 @@ export default function SendAuth({ tokenID }: Props) {
...
+
+        // Redirect
+        uToken
+          ? push("/")
+          : push(
+              `/transaction/${transaction.id}?back=${encodeURIComponent("/")}`
+            );
+
+        // remove wallet from memory
+        freeDecryptedWallet(keyfile);
```
Comment (7i7o) [inline src/routes/popup/send/auth.tsx:251]: We are not calling `freeDecreyptedWallet(keyfile)` if the tx fails

**Review by 7i7o — CHANGES_REQUESTED** [review]
Just a minor fix and it's ready

**Review by elehma4 — APPROVED** [review]
tACK 🚀 LGTM

**Review by 7i7o — APPROVED** [review]
tACK
LGTM! 🚀

**Pull request merged**